src/list/List.tsx

```diff
@@ -36,16 +36,18 @@ export interface ListProps extends HTMLProps<HTMLUListElement> {
    * @default false
    */
   interactive?: boolean;
+  className?: string;
 }
 export function List({
   children,
   listSize = 'default',
   interactive = false,
   noPadding = false,
+  className,
 }: ListProps) {
   return (
     <ul
-      className="ac-list"
+      className={`ac-list${className !== undefined ? ` ${className}` : ''}`}
```

Comment:
```suggestion
      className={`ac-list${className ? ` ${className}` : ''}`}
```
can rely on truthiness to catch null, undefined, and ''